--- libao-0.8.8/src/plugins/pulse/ao_pulse.c.orig	2007-05-24 10:19:07.000000000 +0100
+++ libao-0.8.8/src/plugins/pulse/ao_pulse.c	2008-07-05 21:44:47.000000000 +0100
@@ -75,7 +75,7 @@
     const char *fn;
     struct pa_simple *s;
     static const struct pa_sample_spec ss = {
-        .format = PA_SAMPLE_S16LE,
+        .format = PA_SAMPLE_S16NE,
         .rate = 44100,
         .channels = 2
     };
